win/par2/README_par2cmdline.txt

```diff
@@ -0,0 +1,336 @@
+par2cmdline is a PAR 2.0 compatible file verification and repair tool.
```

Comment:
This file should be called "Licence_par2cmdline.txt" and the other one (for multi-core) "Licence_par2.txt". "COPYING" should be renamed to "GPL2.txt".
The x64 folder has the same issue.